src/com/jdh/microcraft/entity/furniture/EntityChest.java

```diff
@@ -29,6 +30,12 @@ public boolean hit(Entity e) {
 
         return false;
     }
+    @Override
+    protected List<ItemStack> getDrops() {
+        List<ItemStack> chest_items = this.inventory.stacks;
```

Comment:
You should use camelCase for variable names in Java.

Reply:
@lodenrogue fixed in newest commit